api/descriptions/policies/compliance_ci_images_put.md

```diff
@@ -0,0 +1,49 @@
+Updates the compliance policy for images in your continuous integration (CI) pipeline.
```

Comment:
@KICKdesigns Small tweak to this sentence:

...for images built in your continuous...